common/views/components/PopupDialog/PopupDialog.tsx

```diff
@@ -180,7 +180,15 @@ const PopupDialog: FunctionComponent<Props> = ({ document }: Props) => {
   const dialogWindowRef = useRef<HTMLDivElement>(null);
   const { isKeyboard } = useContext(AppContext);
 
+  function hasPersistentState() {
+    const cardiganHosts = ['localhost:9001', 'cardigan.wellcomecollection.org'];
+
+    return cardiganHosts.includes(window.location.host);
+  }
+
   function hidePopupDialog() {
+    if (hasPersistentState()) return;
```

Comment:
could we use this function to set the state once, rather than calling it each time we need to know

Reply:
I'll have a look – I think I'd ruled it out, but maybe I was wrong to